docs/application/toc_application.md

```diff
@@ -621,6 +621,8 @@
 ####### [Properties](/application/native/guides/ui/dali/properties.md)
 ####### [Multi-threaded Architecture](/application/native/guides/ui/dali/multi-threaded.md)
 
+##### TZSH
```

Comment:
To add new page, please indicate the location of the added page in **toc_all.md**, not toc_application.md.
We list up Tizen document pages under /application in toc_all.md.

According to the update in application/native/guides/ui/overview.md,
I guess TZSH page should be located under ui for native, same level to EFL and DALi, after DALi.

If my assumption is correct, the example for the update in toc_all.md is like below.

![image](https://user-images.githubusercontent.com/9958374/53860972-0a76bf00-4026-11e9-9489-1875cf3c3442.png)

I'm not sure whether there will be more pages for TZSH or not.
If we'll have only one page for TZSH, (so if we'll not have overview or introduction page for TZSH),
we can write in one line as I did in the above screenshot.

If there are additional pages, we can set the structure as @gl-lee's.

Reply:
Thanks. :)
I've rollbacked a change of docs/application/toc_application.md and added TZSH link element to docs/application/toc_all.md file as your guide.

Reply:
@gl-lee  thanks 
@mijin85cho  thanks for clarifying the doubt.